syntax = "proto2";
package rare.protos;

message LabelMap {
  optional CharacterSet character_set = 1;
  optional int64 label_offset = 2 [default=0];
  optional int64 unk_label = 3;
}

message CharacterSet {
  oneof source_oneof {
    string text_file = 1 [default=""];
    string text_string = 2 [default=""];
  }
  optional string delimiter = 3 [default=""];
}
